kqueen/utils/logger_config.yml

```diff
@@ -33,6 +33,14 @@ handlers:
         backupCount: 20
         encoding: utf8
 
+    flask_file_handler:
+        class: logging.handlers.RotatingFileHandler
+        level: DEBUG
+        formatter: standard
+        filename: /tmp/kqueen_flask.log
```

Comment:
Pay attention, than in future we need to change dirs to /var/log/kqueen-api
due my changes